### ISSUE TITLE
Is it possible to override the values of global addons

### ISSUE THREAD

**Issue by amittkSharma** [tuchk4/storybook-readme#64]
Hi Team,

Is it possible to override the value of global add-ons, within the story specific values?

Thanks in advance
Amit

**Comment by tuchk4** [comment]
What value do you want to override? Could you please provide more details?

**Comment by amittkSharma** [comment]
I want to override the ReadMe file set in the addDecorator in the config.js. And I want to override this ReadMe set in config.js by the story specific ReadMe.

Thanks in advance.

**Comment by tuchk4** [comment]
Sorry but I didn't catch you. Can you please provide an example what you want to do?

**Comment by amittkSharma** [comment]
In my config.js I am using addDecorator with readme addon and providing it with a default Readme file (DefaultReadMe). Now I am want to override DefaultReadMe in the stories if the story has its own readme (StoryReadme). 

e.g. In my config.js I have a DefaultReadme stating "No Information about the component" now what will happen every story will get the same Readme (DefaultReadme) stating: "No information about the component". Now if any story has it's specific readme (StoryReadme) then I want to replace DefaultReadme with StoryReadme for that specific story,

I hope you will understand the example.

Thanks in advance

**Comment by tuchk4** [comment]
How do you add the default readme?

**Comment by amittkSharma** [comment]
Below is the code for config.js

```
import { withReadme  } from 'storybook-readme'
import * as DefaultReadme from '../DefaultReadme.md'

addDecorator(withReadme(DefaultReadme ))
```

and what I want is that in the story 

```
storiesOf('Test Story A', module)
  .addDecorator(withReadme (StoryReadme))
```

But I want as the story has its own readme file so the default one should be overridden.

Thanks

**Comment by tuchk4** [comment]
Going to implement this feature in next release

**Comment by amittkSharma** [comment]
Thanks a lot.

**Comment by tuchk4** [comment]
This FR released in `4.0.2`
